src/Uno.UI.RemoteControl.VS/EntryPoint.cs

```diff
@@ -310,7 +310,7 @@ private async Task EnsureServerAsync()
 			var pipeGuid = Guid.NewGuid();
 
 			var hostBinPath = Path.Combine(_toolsPath, "host", $"net{version}.0", "Uno.UI.RemoteControl.Host.dll");
-			var arguments = $"\"{hostBinPath}\" --httpPort {_remoteControlServerPort} --ppid {System.Diagnostics.Process.GetCurrentProcess().Id} --ideChannel \"{pipeGuid}\"";
+			var arguments = $"\"{hostBinPath}\" --httpPort {_remoteControlServerPort} --ppid {System.Diagnostics.Process.GetCurrentProcess().Id} --ideChannel \"{pipeGuid}\" --solution \"{_dte.Solution.FullName}\"";
```

Comment:
@spouliot @Takoooooo we need to add support of this new `solution` parameter in VS code and Rider extensions.

Note: we should also make sure to start the dev-server only once per solution.

Reply:
@dr1rrb This `--solution` looks like a new argument added inside this PR. How will the current (and older) version(s) of the tool react if that extra argument is given ?

Reply:
@spouliot Argument will be ignored.